lib/exandra/connection.ex

```diff
@@ -822,6 +821,11 @@ defmodule Exandra.Connection do
     "DROP #{quote_name(name)}"
   end
 
+  defp table_name(sources, idx) do
```

Comment:
We never seem to use `idx`. Why are we not defining `table_name/1` and hard-coding `elem(sources, 0)` on the line below? 🤔

Reply:
That's what I started with, I just had the doubt if we could possibly need to pick another table from sources, but from what I understand `sources` can currently be only a 1-tuple given joins and subqueries are not supported. I'm going to drop `idx` and hardcode `0`.